src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -449,9 +456,10 @@
                 donationMatchingRule: [],
                 donationDateRange: '',
                 postProcessClass: '',
-
+                processUsingScheduledJob: false,
```

Comment:
This shouldn't be here - we've removed all references to processUsingScheduledJob in BGE.